src/main/java/com/mraof/minestuck/data/MinestuckEnUsLanguageProvider.java

```diff
@@ -2068,34 +2068,34 @@ protected void addTranslations()
 		addSubtitles("fmajor_electric_autoharp_stroke", "F Major plays on electric autoharp");
 		addSubtitles("gmajor_electric_autoharp_stroke", "G Major plays on electric autoharp");
 		
-		addEntitySubtitles(MSEntityTypes.NAKAGATOR, "ambient", "Nakagator naks");
-		addEntitySubtitles(MSEntityTypes.NAKAGATOR, "hurt", "Nakagator hurts");
-		addEntitySubtitles(MSEntityTypes.NAKAGATOR, "death", "Nakagator dies");
-		addEntitySubtitles(MSEntityTypes.IGUANA, "ambient", "Iguana thips");
-		addEntitySubtitles(MSEntityTypes.IGUANA, "hurt", "Iguana hurts");
-		addEntitySubtitles(MSEntityTypes.IGUANA, "death", "Iguana dies");
-		addEntitySubtitles(MSEntityTypes.SALAMANDER, "ambient", "Salamander blubs");
-		addEntitySubtitles(MSEntityTypes.SALAMANDER, "hurt", "Salamander hurts");
-		addEntitySubtitles(MSEntityTypes.SALAMANDER, "death", "Salamander dies");
-		addEntitySubtitles(MSEntityTypes.TURTLE, "hurt", "Turtle hurts");
-		addEntitySubtitles(MSEntityTypes.TURTLE, "death", "Turtle dies");
-		addEntitySubtitles(MSEntityTypes.FROG, "ambient", "Frog croaks");
-		addEntitySubtitles(MSEntityTypes.FROG, "hurt", "Frog hurts");
-		addEntitySubtitles(MSEntityTypes.FROG, "death", "Frog dies");
-		addEntitySubtitles(MSEntityTypes.IMP, "ambient", "Imp screeches");
-		addEntitySubtitles(MSEntityTypes.IMP, "hurt", "Imp hurts");
-		addEntitySubtitles(MSEntityTypes.IMP, "death", "Imp dies");
-		addEntitySubtitles(MSEntityTypes.OGRE, "ambient", "Ogre grunts");
-		addEntitySubtitles(MSEntityTypes.OGRE, "hurt", "Ogre hurts");
-		addEntitySubtitles(MSEntityTypes.OGRE, "death", "Ogre dies");
-		addEntitySubtitles(MSEntityTypes.BASILISK, "ambient", "Basilisk hisses");
-		addEntitySubtitles(MSEntityTypes.BASILISK, "hurt", "Basilisk hurts");
-		addEntitySubtitles(MSEntityTypes.BASILISK, "death", "Basilisk dies");
-		addEntitySubtitles(MSEntityTypes.LICH, "ambient", "Lich wispers");
-		addEntitySubtitles(MSEntityTypes.LICH, "hurt", "Lich hurts");
-		addEntitySubtitles(MSEntityTypes.LICH, "death", "Lich dies");
-		addEntitySubtitles(MSEntityTypes.GICLOPS, "ambient", "Giclops wants to eat'cha");
-		addEntitySubtitles(MSEntityTypes.GICLOPS, "hurt", "Giclops hurts");
-		addEntitySubtitles(MSEntityTypes.GICLOPS, "death", "Giclops dies");
+		addEntitySubtitles(MSEntityTypes.NAKAGATOR.get(), "ambient", "Nakagator naks");
```

Comment:
Can you add an overload to `addEntitySubtitles()` that takes a supplier?

Reply:
Let me know if you think what I did is appropriate